capsules/src/screen.rs

```diff
@@ -563,23 +460,53 @@ impl<'a> SyscallDriver for Screen<'a> {
             2 => self.enqueue_command(ScreenCommand::SetPower(data1 != 0), process_id),
             // Set Brightness
             3 => self.enqueue_command(ScreenCommand::SetBrightness(data1), process_id),
-            // Invert On
-            4 => self.enqueue_command(ScreenCommand::InvertOn, process_id),
-            // Invert Off
-            5 => self.enqueue_command(ScreenCommand::InvertOff, process_id),
```

Comment:
I suggest preserving these calls for backward compatibility, at least until we update the libtock-c API. You coud add a comment here saying that they will be removed as soon as the user space API is updated.

Reply:
Done.